### PR TITLE
Added scraper for evozone.com and mcro.tech, evozone.py , mcro.py

### PR DESCRIPTION
**Pull request by TheMrM**

### CODE REVIEW

**Comment by TheMrM** [comment]
On mcro.py the location extracted is output as "location": "Cluj-Napoca, Romania | Remote" . Will be a valid location ?
location

File: sites/mcro.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    
+
+    # Concatenate base_url with job_link
+    full_job_link = base_url + "/" + job_link
+
+    final_jobs.append({
+        'id': str(uuid.uuid4()),
+        'job_link': full_job_link,
+        'job_title': job_title,
+        'location': location,
```
Comment (lalalaurentiu) [inline sites/mcro.py:33]: Location must contain only city, and key is city not location
```suggestion
        'city':'Mioveni',
```

File: sites/mcro.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        'company': company
+    })
```
Comment (lalalaurentiu) [inline sites/mcro.py:35]: you forgot country
```suggestion
          'country':'Romania'
    })
```

File: sites/mcro.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+soup = BeautifulSoup(html_markup, 'html.parser')
+
+jobs = soup.find_all('a', class_='job')
+
+final_jobs = []
+
+for job in jobs:
+    job_link = job['href']
+    job_title = job.find('h4').get_text(strip=True)
+    location = job.find('div', class_='job__description--location').find('h5').get_text(strip=True)
```
Comment (lalalaurentiu) [inline sites/mcro.py:23]: Show only city
```suggestion
    location = job.find('div', class_='job__description--location').find('h5').get_text(strip=True).split(',')[0]
```

File: sites/mcro.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        'company': company
+    })
+
+api_key = os.environ.get('Grasum_Key')
+
+clean_data = requests.post('https://api.peviitor.ro/v4/clean/', headers={'Content_Type': 'application/x-www-form-urlencoded', 'apikey': api_key}, data={'company': company})
+
+update_data = requests.post('https://api.peviitor.ro/v4/update/', headers={'Content_Type': 'application/json', 'apikey': api_key}, data=json.dumps(final_jobs))
+
+requests.post('https://api.peviitor.ro/v1/logo/add/', headers={'Content_Type': 'application/json'}, data=json.dumps([{'id': company, 'logo': 'https://www.evozon.com/wp-content/uploads/2021/03/Group-813.svg'}]))
```
Comment (lalalaurentiu) [inline sites/mcro.py:43]:
```suggestion
requests.post('https://api.peviitor.ro/v1/logo/add/', headers={'Content_Type': 'application/json'}, data=json.dumps([{'id': company, 'logo': 'mcro logo not evozone'}]))
```

**Review by lalalaurentiu — CHANGES_REQUESTED** [review]

**Review by lalalaurentiu — APPROVED** [review]

**Pull request merged**